src/app/common/resolvers/dataset-view/dataset-settings/dataset-settings-var-and-secrets.resolver.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { inject } from "@angular/core";
+import { ResolveFn } from "@angular/router";
+import { AppConfigService } from "src/app/app-config.service";
+import { SettingsTabsEnum } from "src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.model";
+import { DatasetEvnironmentVariablesService } from "src/app/dataset-view/additional-components/dataset-settings-component/tabs/variables-and-secrets/dataset-evnironment-variables.service";
+import { VariablesAndSecretsData } from "src/app/dataset-view/dataset-view.interface";
+import { DatasetService } from "src/app/dataset-view/dataset.service";
+import { DatasetSubscriptionsService } from "src/app/dataset-view/dataset.subscriptions.service";
+import ProjectLinks from "src/app/project-links";
+import { NavigationService } from "src/app/services/navigation.service";
+import { datasetSettingsActiveSectionResolver } from "../dataset-settings-active-section.resolver";
+import { combineLatest, EMPTY, map, of, switchMap } from "rxjs";
+import { DatasetMetadata, ViewDatasetEnvVarConnection } from "src/app/api/kamu.graphql.interface";
+import { isSettingsTabAccessibleHelper } from "src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.helpers";
+
+export const datasetSettingsVarAndSecretsResolver: ResolveFn<VariablesAndSecretsData> = (route, state) => {
+    const evnironmentVariablesService = inject(DatasetEvnironmentVariablesService);
+    const datasetService = inject(DatasetService);
+    const datasetSubService = inject(DatasetSubscriptionsService);
+    const navigationService = inject(NavigationService);
+    const appConfigService = inject(AppConfigService);
+    const activeTab = datasetSettingsActiveSectionResolver(route, state) as SettingsTabsEnum;
```

Comment:
You put this resolver into a very specific routing URL, so you perfectly know what your tab is.

Reply:
Removed `datasetSettingsActiveSectionResolver`